oscar_odin/mappings/context.py

```diff
@@ -51,10 +54,11 @@ class ModelMapperContext(dict):
     one_to_many_items = None
     attribute_data = None
     identifier_mapping = None
+    product_identity_list = None
```

Comment:
I don't think this should be called product_identity_list as it's more generic than the products